persistence/src/commonMain/db_user/migrations/81.sqm

```diff
@@ -0,0 +1,169 @@
+DROP INDEX IF EXISTS message_expire_after_millis_index;
+
+DROP VIEW IF EXISTS MessageDetailsView;
+
+ALTER TABLE Message DROP COLUMN self_deletion_start_date;
+
+CREATE INDEX message_expire_after_millis_index ON Message(expire_after_millis);
```

Comment:
question: is  creating a new index is a bit faster if there it is already included in another composite index?
does it make sense to create the new index before deleting 
```suggestion
CREATE INDEX message_expire_time_index ON Message(expire_after_millis);

DROP INDEX IF EXISTS message_expire_after_millis_index;
DROP VIEW IF EXISTS MessageDetailsView;
ALTER TABLE Message DROP COLUMN self_deletion_start_date;
```

Reply:
Hmm, at first glance it doesn't look like it takes advantage of another index when creating one, `EXPLAIN` shows exactly the same actions when there's already other composite index and when not, but the only disadvantage of changing the order like so is that we just need to use different name for the new index so I applied this change just in case.